service/modules/process_result.py

```diff
@@ -0,0 +1,194 @@
+"""Process result file module"""
+import os
+import json
+import datetime
+import logging
+import ast
+import jsend
+import falcon
+import pytz
+import pysftp
+import sendgrid
+import sentry_sdk
+import pandas as pd
+from .permit_applications import PermitApplication
+from ..resources.export import Export
+
+ERROR_GENERIC = "Bad Request"
+ERROR_ACCESS_401 = "Unauthorized"
+
+class ProcessResultFile():
+    """ Class for processing result file """
+    data_file_path = os.path.dirname(__file__) + '/exported_data/'
+
+    def on_get(self, req, resp):
+        """ on get request """
+        try:
+            if req.params['token'] != os.environ.get('EXPORT_TOKEN') and req.params['token'] != os.environ.get('ACCESS_KEY'):
```

Comment:
Can't understand what is going on here.  It seems weird to check token != EXPORT_TOKEN AND token != ACESS_KEY. the only way this statement would be true is if the EXPORT_TOKEN and ACESS_KEY are the same value, and token is set to it.